project_name/schemas/my_table_schema.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
This seems project specific, not really template generic but I might be mistaken?